node/core/av-store/Cargo.toml

```diff
@@ -19,6 +19,8 @@ polkadot-node-subsystem-util = { path = "../../subsystem-util" }
 polkadot-overseer = { path = "../../overseer" }
 polkadot-primitives = { path = "../../../primitives" }
 
+sc-service = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
```

Comment:
```suggestion
sc-service = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Reply:
That comes from https://github.com/paritytech/polkadot/pull/1795/commits/a4642e117e89f22e9301c5b22d970cbbe7b60727 which fixes [this CI failure](https://gitlab.parity.io/parity/polkadot/-/jobs/689145).